File: Algo/max.js
function return_max(arr){
    var keep = 0;
    while (arr.length > 1){
        if (arr[0] >= arr[(arr.length)-1]){
            arr.pop()
        }
        else {
            var x = arr[(arr.length)-1]
            arr[(arr.length)-1] = arr[0]
            arr[0] = x
            arr.pop()
            }
    keep = arr[0]
    }
    return keep
}

test = return_max([1,5,-300,16,-2,19,12])
console.log(test)

File: Algo/max.py
def arr_max(arr):
    """takes the values of an array and returns the max"""
    keep = []
    trash = []
    for value in arr:
        if len(arr) == 1:
            break
        elif value >= arr[-1]:
            keep.append(value.pop())
        else:
            trash.append(value.pop())
    print(keep)
    print(trash)
    print(arr)

arr_max([1,4,3,9,10,-2,4])
